Pomile/GeoMarket
# [#172908484] database migration

#### What does this PR do?
Migrate database
#### Description of Task to be completed?
- [x] Install packages
- [x] Create .sequelizerc
- [x] Create database configuration file
- [x] Create models
- [x] Create migration files
- [x] Create hash helper
- [x] Create seeders data
- [x] Create seeders
#### How should this be manually tested?
1. Pull this branch `git pull origin ch-database-migration-172908484`
2. Install prerequisite(NodeJs, Postgresql)
3. Create database
4. Create .env file that looks like the following
```
DB_USERNAME=poxxxx
DB_PASSWORD=xxxxx
DB_DEV_NAME=xxxxx
DB_TEST_NAME=xxxxxx
DB_PORT=xxxx
DB_HOST=xxxx
PORT=xxxx
```
5. Run `npm install`
6. Run `npx sequelize-cli db:migrate`
7. Run `npx sequelize-cli db:seed:all`
8. Check seeded data through command line(`e.g select * from users;`) or pgAdmin


#### What are the relevant pivotal tracker stories?
[#172908484](https://www.pivotaltracker.com/story/show/172908484)
#### Any background context you want to add?
NIL
#### Screenshots
NIL
